app/src/main/java/com/amaze/filemanager/utils/RootUtils.java

```diff
@@ -41,27 +58,41 @@
     private static String mountFileSystemRW(String path) throws ShellNotRunningException {
         String command = "mount";
         ArrayList<String> output = RootHelper.runShellCommandToList(command);
-        String mountPoint = "", types = null;
+        String mountPoint = "", types = null, mountArgument = null;
         for (String line : output) {
             String[] words = line.split(" ");
 
-            if (path.contains(words[2])) {
+            // mount command output for older Androids
+            // <code>/dev/block/vda /system ext4 ro,seclabel,relatime,data=ordered 0 0</code>
+            String mountPointOutputFromShell = words[1];
+            String mountPointFileSystemTypeFromShell = words[2];
+            String mountPointArgumentFromShell = words[3];
+            if(Build.VERSION.SDK_INT > Build.VERSION_CODES.M) {
```

Comment:
Not >=?

Reply:
Good catch. Will clarify this before merging.